src/registrar/management/commands/utility/terminal_helper.py

```diff
@@ -203,8 +203,21 @@ def log_script_run_summary(
         if log_header is None:
             log_header = "============= FINISHED ==============="
 
+        if skipped_header is None:
+            skipped_header = "----- SOME DATA WAS INVALID (NEEDS MANUAL PATCHING) -----"
+
+        # Give the user the option to see failed / skipped records if any exist.
+        debug_anyway = False
```

Comment:
Is it possible to update the name to be something more specific? I didn't quite know what you meant by debug_anyway.